### PR TITLE
VDS-486: Upload part icon

### PR DESCRIPTION
**Pull request by asvishnyakov**

### CODE REVIEW

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            name: "platform.commands.confirm",
+            icon: 'fa fa-check',
+            executeMethod: () => $scope.saveChanges(),
+            canExecuteMethod: () => $scope.canSave()
+        }];
+
+        function initialize () {
+            let uploader = $scope.uploader = new FileUploader({
+                scope: $scope,
+                //headers: { Accept: 'application/json' }
```
Comment (ilyawzrd) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.js]: Remove commented code if it's not used

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.tpl.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                        <table class="table preview" ng-if-end>
+                            <tbody>
+                                <tr class="table-item">
+                                    <td class="table-col __product-img">
+                                        <img ng-src="{{blade.currentEntity.imgSrc ? blade.currentEntity.imgSrc : ''}}">
+                                    </td>
+                                    <td class="table-col">
+                                        <div class="table-t">{{ blade.currentEntity.name ? blade.currentEntity.name : ('demoSolutionFeaturesModule.blades.upload-part-icon.preview.image.title' | translate) }}</div>
+                                        <div class="table-descr">
+                                            {{ 'demoSolutionFeaturesModule.blades.upload-part-icon.preview.image.description' | translate: { count: blade.currentEntity.partItems ? blade.currentEntity.partItems.length : 0 } }}
```
Comment (ilyawzrd) [inline src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.tpl.html]: Can we add a localization for single partItem case?

**Review by ilyawzrd — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.js (hunk 1 above)
Comment (asvishnyakov) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.js]: Fixed

File: src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.tpl.html (hunk 2 above)
Comment (asvishnyakov) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.tpl.html]: Fixed, but in another way (too additional code : )

Comment (pushnitsa) [reply src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/upload-part-icon.tpl.html]: I think we should use single and plural description. To reduce code base, we could use ternary operator.
Like in example: https://stackoverflow.com/questions/46350494/angular-translate-filter-in-ternary-operator

**Review by pushnitsa — DISMISSED** [review]

**Review by ilyawzrd — DISMISSED** [review]

**Review by pushnitsa — APPROVED** [review]

**Pull request merged**